use charm-dev blueprint for multipass (#84)
canonical/jaas-documentation: tutorial/deploy_jaas_microk8s.rst

Prerequisites
-------------
- A workstation, e.g., a laptop, that has sufficient resources to launch a virtual machine with 4 CPUs, 8 GB RAM, and 50 GB disk space.

Set up an isolated test environment
-----------------------------------

Set up an isolated test environment with Multipass and the `charm-dev` blueprint, which will provide all the necessary tools and configuration for the tutorial (a localhost machine cloud and Kubernetes cloud, Juju, etc.). 

On your machine, install Multipass and use it to set up an Ubuntu virtual machine (VM) called `my-juju-vm` from the `charm-dev` blueprint. 

.. note::
    See more: `Set things up (automatically) <https://canonical-juju.readthedocs-hosted.com/en/latest/user/howto/manage-your-deployment/manage-your-deployment-environment/#manage-your-deployment-environment>`__  

>     Note: This document also contains a manual path, using which you can set things up without the Multipass VM or the `charm-dev` blueprint. However, please note that the manual path may yield slightly different results that may impact your experience of this tutorial. 

Reviewer: Should we not make this more evident since in our case things don't yet work without Multipass?

    For best results we strongly recommend the automatic path, or else suggest that you follow the manual path in a way that stays very close to `the definition of the charm-dev blueprint <https://github.com/canonical/multipass-blueprints/blob/e270a76093aad7b178ce0df5b7aa00e9dfd9b054/v1/charm-dev.yaml>`__.

Open a shell in the VM:

.. code:: bash
    
    multipass shell my-juju-vm

> Make sure MicroK8s is correctly setup:

Reviewer 2: ```suggestion
Make sure MicroK8s is correctly set up:
```

.. code:: bash
    
    # enable necessary add-ons
    sudo microk8s dns ingress host-access

> Then install some handy tools:

Reviewer 2: ```suggestion
Then install some handy tools for querying content:
```

Basically just something that would prevent a question mark in the reader's head.

.. code:: bash

    sudo apt install jq
    sudo snap install yq

> You are now set up to deploy JAAS.

Reviewer 2: ```suggestion
You are now all set and ready to deploy JAAS.
```
